scen_def/SXC_System.cfg

```diff
@@ -127,13 +127,28 @@
 
     {SXC_ENEMY_LEADER_CHANGE_ATTACK 6 0 (set_type=cold
     set_description="frosty longsword")}
+    {SXC_ENEMY_LEADER_ADD_ABILITY 7 {ABILITY_SXC_DEATHAURAX 100}}
+    {SXC_ENEMY_LEADER_HALO 7 {SXC_DEATHAURA_HALO}}
     {SXC_ENEMY_LEADER_CHANGE_ARMOR 7 arcane 40}
     {SXC_ENEMY_LEADER_CHANGE_ARMOR 7 cold 30}
     {SXC_ENEMY_LEADER_CHANGE_ARMOR 7 fire 90}
     {SXC_ENEMY_LEADER_CHANGE_ARMOR 7 pierce 50}
     {SXC_ENEMY_LEADER_CHANGE_ATTACK 7 0 (set_type=fire
     set_description="flaming sword")}
-
+    
+    {SXC_ENEMY_LEADER_ADD_ABILITY 8 {ABILITY_SXC_DARKAURA}}
+    {SXC_ENEMY_LEADER_HALO 8 {SXC_DARKAURA_HALO}}
+    {SXC_ENEMY_LEADER_SET_SPECIALS 8 3 append {WEAPON_SPECIAL_SXC_EVADE} {BLANK}}
+    {SXC_ENEMY_LEADER_CHANGE_ATTACK 8 3 (increase_attacks=1)}
```

Comment:
I recall ai8 and ai9 in SXC System already being pretty tough, I'm hesitant to make them tougher.